### PR TITLE
fix(i18n): translated translations.json to Chinese

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

File: client/i18n/locales/chinese/translations.json [hunk 1]
```diff
@@ -1,444 +1,444 @@
...
-    "big-heading-1": "Learn to code at home.",
-    "big-heading-2": "Build projects.",
-    "big-heading-3": "Earn certifications.",
-    "h2-heading": "Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including:",
-    "hero-img-description" : "freeCodeCamp students at a local study group in South Korea.",
-    "as-seen-in": "As seen in:",
+    "big-heading-1": "在家学习编程",
+    "big-heading-2": "构建项目",
+    "big-heading-3": "获得认证",
+    "h2-heading": "自 2014 年以来，已有超过 40,000 名 freeCodeCamp 学员入职科技公司，例如：",
```
Comment (moT01) [inline client/i18n/locales/chinese/translations.json:74]:
```suggestion
    "h2-heading": "自 2014 年以来，已有超过 40,000 名 freeCodeCamp.org 学员入职科技公司，例如：",
```

Not sure, english has it.

File: client/i18n/locales/chinese/translations.json [hunk 2]
```diff
@@ -1,444 +1,444 @@
...
-    "you-not-public": "You have not made your portfolio public.",
-    "username-not-public": "{{username}} has not made their portfolio public.",
-    "you-change-privacy": "You need to change your privacy setting in order for your portfolio to be seen by others. This is a preview of how your portfolio will look when made public.",
-    "username-change-privacy": "{{username}} needs to change their privacy setting in order for you to view their portfolio.",
+    "you-not-public": "你尚未把你的作品集设置为公开。",
+    "username-not-public": "{{username}} 未把作品集设置为公开",
+    "you-change-privacy": "你需要更改你的隐私设置，才能让他人看到你的作品集。这是你的作品集公开时候的预览。",
+    "username-change-privacy": "{{username}} 需要修改隐私设置，你才能看到其作品集。",
     "supporter": "Supporter",
     "contributor": "Top Contributor",
```
Comment (moT01) [inline client/i18n/locales/chinese/translations.json:190]: No translations or something for these two I take it?

File: client/i18n/locales/chinese/translations.json [hunk 3]
```diff
@@ -1,444 +1,444 @@
...
-    "msg-38": "That does not appear to be a valid challenge submission",
-    "msg-39": "You have not provided the valid links for us to inspect your work.",
-    "msg-40": "No social account found",
-    "msg-41": "Invalid social account",
-    "msg-42": "No {{website}} account associated",
-    "msg-43": "You've successfully unlinked your {{website}}",
-    "msg-44": "Check if you have provided a username and a report",
-    "msg-45": "A report was sent to the team with {{email}} in copy"
+    "msg-1": "申请认证之前，你必须先接受我们的《学术诚信条例》",
+    "msg-2": "出现了一些奇怪的情况。如果再出现这种情况，请考虑在 https://github.com/freeCodeCamp/freeCodeCamp/issues/new 提 issue",
```
Comment (moT01) [inline client/i18n/locales/chinese/translations.json:386]:
```suggestion
    "msg-2": "出现了一些奇怪的情况。如果再出现这种情况，请考虑在 https://github.com/freeCodeCamp/freeCodeCamp/issues/new 提",
```

Doesn't seem right, not sure.

File: client/i18n/locales/chinese/translations.json [hunk 4]
```diff
@@ -1,444 +1,444 @@
...
+    "msg-14": "主题已设置",
+    "msg-15": "你的主题已更新！",
+    "msg-16": "用户名已和此账号关联",
+    "msg-17": "用户名已和别的账号关联",
+    "msg-18": "我们已更新你的用户名为 {{username}}",
+    "msg-19": "你的账号未退出，请稍后重试。",
+    "msg-20": "链接中的邮箱格式不正确",
+    "msg-21": "有点不对劲。请刷新链接再登录/注册",
+    "msg-22": "你点击的链接似乎已过期，请刷新链接再登录",
+    "msg-23": "成功了！你已经登录账号。Happy Coding!",
```
Comment (moT01) [inline client/i18n/locales/chinese/translations.json:407]:
```suggestion
    "msg-23": "成功了！你已经登录账号。",
```

Not sure.

**Review by moT01 — COMMENTED** [review]
I pointed out a few things that don't look right to me. I can't read Chinese, so I'm not sure. Want to take a quick look @miyaliu666? If you like the suggestion, you can click the `commit suggestion` button next to it to add it.

File: client/i18n/locales/chinese/translations.json (hunk 2 above)
Comment (miyaliu666) [reply client/i18n/locales/chinese/translations.json:190]: I suggest we just keep the special titles as English, for consistency with the global community.

File: client/i18n/locales/chinese/translations.json (hunk 3 above)
Comment (miyaliu666) [reply client/i18n/locales/chinese/translations.json:386]: It's ok to use "issue" directly in the translation. And it shouldn't be omitted.

File: client/i18n/locales/chinese/translations.json (hunk 4 above)
Comment (miyaliu666) [reply client/i18n/locales/chinese/translations.json:407]: emmm I always use "Happy Coding" in articles. But ok, we can delete it here.

File: client/i18n/locales/chinese/translations.json (hunk 2 above)
Comment (raisedadead) [reply client/i18n/locales/chinese/translations.json]: I think we should leave the titles as is.

**Pull request merged**